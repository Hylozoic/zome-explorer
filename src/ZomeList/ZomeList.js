import React from 'react'
import Zome from '../Zome'

export default function ZomeList ({ callZome, zomes }) {
  return <div className='zome-list'>
    {zomes.map(zome => <Zome zome={zome} key={zome.name} callZome={callZome} />)}
  </div>
}