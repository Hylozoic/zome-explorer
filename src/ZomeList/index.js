import ZomeList from './ZomeList'
export default ZomeList