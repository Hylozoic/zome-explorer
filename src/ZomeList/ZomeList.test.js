import ZomeList from './ZomeList'
import { shallow } from 'enzyme'
import React from 'react'

it('matches the last snapshot', () => {
  const props = {
    zomes: [
      {name: 'zome one'},
      {name: 'zome two'}
    ],
    callZome: name => () => {},    
  }  
  const wrapper = shallow(<ZomeList {...props} />)
  expect(wrapper).toMatchSnapshot()
})
