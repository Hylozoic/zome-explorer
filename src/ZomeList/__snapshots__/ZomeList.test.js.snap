// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the last snapshot 1`] = `
<div
  className="zome-list"
>
  <div
    className="zome-list-header"
  >
    Zomes
  </div>
  <Zome
    callZome={[Function]}
    key="zome one"
    zome={
      Object {
        "name": "zome one",
      }
    }
  />
  <Zome
    callZome={[Function]}
    key="zome two"
    zome={
      Object {
        "name": "zome two",
      }
    }
  />
</div>
`;
